internal/entity/activity.go

```diff
@@ -36,32 +34,30 @@ var AllActivityStatusValues = []string{
 }
 
 type Activity struct {
+	Metadata
 	Id        int64               `json:"id"`
 	Status    ActivityStatusValue `json:"status"`
 	Service   *Service            `json:"service,omitempty"`
 	Issues    []Issue             `json:"issues,omitempty"`
 	Evidences []Evidence          `json:"evidences,omitempty"`
-	CreatedAt time.Time           `json:"created_at"`
-	DeletedAt time.Time           `json:"deleted_at,omitempty"`
-	UpdatedAt time.Time           `json:"updated_at"`
 }
 
 func (a *Activity) GetId() int64 {
 	return a.Id
 }
 
 type ActivityHasIssue struct {
-	ActivityId int64     `json:"activity_id"`
-	IssueId    int64     `json:"issue_id"`
-	CreatedAt  time.Time `json:"created_at"`
-	DeletedAt  time.Time `json:"deleted_at,omitempty"`
-	UpdatedAt  time.Time `json:"updated_at"`
+	Metadata
+	ActivityId int64 `json:"activity_id"`
+	IssueId    int64 `json:"issue_id"`
 }
 
 type ActivityAggregations struct {
+	Metadata
```

Comment:
I don't think we need the Metadata for Aggregations, because they will only contain aggregated information for the model. For example the number of Issues corresponding to that Activity. This means the aggregation is not stored in the database and cannot be modified by a user directly

Reply:
This is also the case for all other Aggregation structs

Reply:
removed also from filters, paginated and list options